Bucket_Sort/Bucket_Sort.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
+	"fmt"
+	"sort"
+)
+
+// Function to sort the array elements using Bucket sort
+func bucket_sort(arr []float64, size int)
+{
+	var max_ele, min_ele float64
+    min_ele = arr[0]
+
+    //Find the minimum and maximum element in the array
+	for i:= 0; i < size; i++ { 
+
+		if arr[i] < min_ele {
```

Comment:
TAB has been used for indentation in this file, Can you use 4 space indentation ?